docker/production/docker-compose.yml

```diff
@@ -28,7 +40,22 @@ services:
       OE_USER: admin
       OE_PASS: pass
     depends_on:
-    - mysql
+      mysql:
+        condition: service_healthy
+    healthcheck:
+      test:
+      - CMD
+      - /usr/bin/curl
+      - --silent
+      - --fail
+      - --show-error
+      - --location
+      - http://localhost/
```

Comment:
For sure we would want https to be the health check on the production docker, right?